app/src/androidTest/kotlin/org/wordpress/aztec/demo/pages/EditLinkPage.kt

```diff
@@ -33,7 +33,17 @@ class EditLinkPage : BasePage() {
     }
 
     fun updateURL(url: String): EditLinkPage {
-        urlField.perform(replaceText(url), ViewActions.closeSoftKeyboard())
+        threadSleep(2000L) // Wait for the page to load
+        try {
+            urlField.perform(click())
+            urlField.perform(replaceText(url), ViewActions.closeSoftKeyboard())
+        } catch (e: RuntimeException) {
+            // If the URL field is not visible, it might be because the keyboard is open.
+            // Close the keyboard and try again.
+            urlField.perform(ViewActions.closeSoftKeyboard())
+            urlField.perform(replaceText(url), ViewActions.closeSoftKeyboard())
+            e.printStackTrace()
+        }
```

Comment:
This is the core change: a few of additional safety checks and delays to make UI tests not flaky on CI.